packages/files-ui/cypress/support/page-objects/authenticationPage.ts

```diff
@@ -12,6 +12,10 @@ export const authenticationPage = {
   // sign in section elements
   loginPasswordButton: () => cy.get("[data-cy=login-password-button]", { timeout: 20000 }),
   loginPasswordInput: () => cy.get("[data-cy=login-password-input]"),
+  signInExplainerContinueButton: () => cy.get("[data-cy=sign-in-explainer-continue-button]", { timeout: 20000 }),
+  signInSetupPasswordInput: () => cy.get("[data-cy=sign-in-password-input]"),
+  signInSetupPasswordVerificationInput: () => cy.get("[data-cy=sign-in-password-verification-input]"),
+  signInSetupPasswordSubmitButton: () => cy.get("[data-cy=sign-in-password-button]"),
```

Comment:
```suggestion
  signInExplainerContinueButton: () => cy.get("[data-cy=button-sign-in-explainer-continue]", { timeout: 20000 }),
  signInSetupPasswordInput: () => cy.get("[data-cy=input-sign-in-password]"),
  signInSetupPasswordVerificationInput: () => cy.get("[data-cy=input-sign-in-password-verification]"),
  signInSetupPasswordSubmitButton: () => cy.get("[data-cy=button-sign-in-password]"),
```
Very minor - I've been putting the element type at the beginning of the string for identifiers, just a small suggestion for consistency

Reply:
I'm fine with that, but just look above, they all finish by "button" or "input", that's what I copied :D

Reply:
I'm fine with that, but just look above, they all finish by "button" or "input", that's what I copied :D

Reply:
@Tbaut 🤦‍♂️ 😂 ah I guess this is a page object that I created before following that approach. 

No worries, lets merge as is and I'll follow up with a separate PR for consistency of the pre-existing ones, there's likely to be others so I'll do a full review.